src/pkg/cli/compose/convert.go

```diff
@@ -310,8 +313,10 @@ func convertPort(port compose.ServicePortConfig) *defangv1.Port {
 				term.Warnf("Published ports are ignored in ingress mode")
 			}
 			pbPort.Mode = defangv1.Mode_INGRESS
-			if pbPort.Protocol == defangv1.Protocol_TCP || pbPort.Protocol == defangv1.Protocol_UDP {
-				term.Warnf("TCP ingress is not supported; assuming HTTP (remove 'protocol' to silence)")
+			if pbPort.Protocol == defangv1.Protocol_TCP {
+				if pbPort.Protocol == defangv1.Protocol_UDP {
```

Comment:
Logic error? Second if is always false

Reply:
Good catch, in fact pbPort.Protocol will never by UDP, as if pbPort.Protocol is only going to be 'udp' if port.Protocol is 'udp, see line 292~293 above. So this whole block can be removed.